pkg/models/vulnerability.go

```diff
@@ -73,7 +73,7 @@ type Affected struct {
 func (a Affected) MarshalJSON() ([]byte, error) {
 	type rawAffected Affected // alias Affected to avoid recursion during Marshal
 	type wrapper struct {
-		Package *Package `json:"package,omitempty" yaml:"package,omitempty"`
+		Package *Package `json:"package,omitempty"`
```

Comment:
Is this no longer needed?

Reply:
In `MarshalJSON` it is unused and falls back to the tag in the main struct